feature-based-molecular-networking/tools/feature-based-molecular-networking/scripts/progenesis_formatter.py

```diff
@@ -69,23 +77,25 @@ def convert_to_feature_csv(input_filename, output_filename):
 
         compound_to_scan_mapping[compound_name] = running_scan
 
-    # Prepared the processed tables
+        # Prepared the processed tables
+
     output_df = pd.DataFrame(output_records)
+    output_df = output_df.drop([x for x in output_df if x.endswith('.1')], 1)
```

Comment:
What is this doing, can you comment?

Reply:
Yes.     Adding this
# We are dropping the Raw abundance columns (that have a suffixe .1) keeping only the Normalized abundance column

Reply:
Can you add this documentation into the README?

Reply:
This is already specified in the README file

Reply:
Sorry I didnt see it in there, which portion denotes that the suffix means raw abundance.

Reply:
Now explicit from the readme